### PR TITLE
Task delete transacciones

### PR DESCRIPTION
**Pull request by AndreeToledo**
#### What's this PR do?
It allows a capturista to delete a transaccion from a family.

#### Where should the reviewer start?
The reviewer should check the view for getting the delete modal
The reviewer should then check the view for actually deleting the transaccion
Then the reviewer should check the form for deleting

#### How should this be manually tested?
Go to the transaccion page of any family
Click the delete button on any transaccion
Click the confirm button
Check that the transaccion doesn't appear and that the economic status table has changed

#### Any background context you want to provide?
This pull request was opened before the tests are completed in order to allow @csssaz and @fernandolobato to check the code before they travel tomorrow

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: jp2_online/templates/captura/transaccion_delete_modal.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<form class="form-horizontal edit_form" method="POST" action="{% url 'captura:delete_transaccion' transaccion.pk %}">
+  {% csrf_token %}
+  {{ delete_form }}
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    <h4 class="modal-title">Borrar integrante</h4>
+  </div>
+  
+  <div class="modal-body">
+    <p>¿Está seguro que desea borrar la transaccion {{ transaccion.observacion }}? Esta acción no se puede revertir.</p>
```
Comment (csssaz) [inline jp2_online/templates/captura/transaccion_delete_modal.html]: transaccion -> transacción

File: familias/forms.py [hunk 2]
```diff
@@ -149,6 +149,7 @@ def clean(self):
         to a real integrante.
         """
         cleaned_data = super(DeleteIntegranteForm, self).clean()
+        print(cleaned_data['id_integrante'])
```
Comment (fernandolobato) [inline familias/forms.py]: Remove

File: indicadores/forms.py [hunk 3]
```diff
@@ -56,3 +56,28 @@ def __init__(self, id_familia, *args, **kwargs):
...
             field.widget.attrs['class'] = 'form-control'
+
+
+class DeleteTransaccionForm(Form):
+    """Form to delete user from dashboard which is used to validate the post information.
+
+    """
+    id_transaccion = IntegerField(widget=HiddenInput())
+
+    def clean(self):
```
Comment (csssaz) [inline indicadores/forms.py]: make sure to test this properly. you can check (pretty much copy and paste) the tests i did for the forms

File: captura/tests.py [hunk 4]
```diff
@@ -240,6 +241,38 @@ def test_update_ingreso_incomplete(self):
...
+        for rendering the modal returns 400 if the request is not ajax.
+        """
+        url = reverse('captura:form_delete_transaccion',
+                      kwargs={'id_transaccion': self.transaccion1.pk})
+        response = self.client.post(url)
+        self.assertEqual(400, response.status_code)
+        response = self.client.get(url)
+        self.assertEqual(400, response.status_code)
+
+    def test_get_modal_delete_integrante(self):
```
Comment (csssaz) [inline captura/tests.py]: change the name

File: indicadores/forms.py [hunk 5]
```diff
@@ -56,3 +56,28 @@ def __init__(self, id_familia, *args, **kwargs):
         self.fields['tutor'].queryset = tutores
         for field_name, field in self.fields.items():
             field.widget.attrs['class'] = 'form-control'
+
+
+class DeleteTransaccionForm(Form):
+    """Form to delete user from dashboard which is used to validate the post information.
+
+    """
+    id_transaccion = IntegerField()
```
Comment (csssaz) [inline indicadores/forms.py]: you should hide this

**Pull request merged**